cmd/sb/template.html

```diff
@@ -1,141 +1,132 @@
-          </td>
-          {{if gt $row.PenaltyTime 0.0}}<td class="penalty">{{$row.PenaltyTime | printf "%.0f"}}</td>{{else}}<td></td>{{end}}
-          {{range $cell := $row.Cells}}
-          <td class="{{$cell.Class}}" title="{{$cell.Title}}">{{$cell.Value}}</td>
-          {{end}}
-        </tr>
-        {{end}}
-      </tbody>
-    </table>
-  </main>
-  <div>
-    The scoreboard and judge submissions are not used for grading.
-    Only code submitted to iLMS count.
-  </div>
-</body>
-
-</html>
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+
+    <title>{{.Homework.Name}}</title>
+    
+    <style>
+      body {
+        font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
+        font-weight: normal;
+        font-size: 14px;
+      }
+      th:first-child, td:first-child {
+        position: sticky;
+        left: 0px;
+        background-color: white;
+        box-shadow: 0px -0.5px rgb(222,226,230);
+      }
+      .table-responsive {
+        padding: 0;
+      }
+      tbody tr:hover th {
+        background-color: rgb(236,236,236);
+      }
+      .table td, th {
+        text-align: center;
+      }
+      .table thead tr th {
+        border-top-width: 1px;
+        border-top: 0px;
+      }
+    </style>
+  </head>
+  
+  <body>
+    <nav class="navbar navbar-light" style="background-color: #FF9800;">
+      <a class="navbar-brand" href="#" style="color: white">{{.Homework.Name}} Scoreboard</a>
+    </nav>
+    <br>
+    
+    <div style="padding: 0px 20px">
+      <div class="container-flux table-responsive">
+        <table id="thetable" class="table table-sm table-hover" data-fixed-columns=true data-fixed-number=1>
+          <thead>
+            <tr>
+              <th scope="col">User</th>
+              <th scope="col">Rank</th>
+              <th scope="col">Passed</th>
+              <th scope="col">Time</th>
+              <th scope="col">Penalty</th>
+              {{range $name := .Homework.Cases}}
+              <th>{{$name}}</th>
+              {{end}}
+            </tr>
+          </thead>
+          <tbody>
+            {{range $row := .Rows}}
+            <tr>
+              <th>{{$row.Submission.User}}</th>
+              <td class="center">{{$row.Rank}}</td>
+              <td class="center">{{$row.NumPassed}}</td>
+              <td>
+                {{$row.TotalTime | printf "%.2f"}}
+              </td>
+              {{if gt $row.PenaltyTime 0.0}}<td class="penalty">{{$row.PenaltyTime | printf "%.0f"}}</td>{{else}}<td></td>{{end}}
```

Comment:
Looks like `class="penalty"` is no longer used

Reply:
I add a `+` symbol before the penalty time in the latest commit, so I keep the `class="penalty"` here.